Upstream svn 38091
fix messed up output of vptovf and pltotf etc
---
 texk/web2c/ChangeLog            |    5 +++++
 texk/web2c/omegaware/ChangeLog  |    5 +++++
 texk/web2c/omegaware/opl2ofm.ch |    1 +
 texk/web2c/omegaware/ovp2ovf.ch |    1 +
 texk/web2c/pltotf.ch            |    1 +
 texk/web2c/vptovf.ch            |    1 +
 6 files changed, 14 insertions(+)

--- texlive-bin.orig/texk/web2c/ChangeLog
+++ texlive-bin/texk/web2c/ChangeLog
@@ -1,3 +1,8 @@
+2015-08-10  Peter Breitenlohner  <peb@mppmu.mpg.de>
+
+	* pltotf.ch, vptovf.ch: Send also real numbers to stderr,
+	completing the change from 2015-01-09 (report from Norbert).
+
 2015-05-21  Karl Berry  <karl@tug.org>
 
 	* TL'15.
--- texlive-bin.orig/texk/web2c/omegaware/ChangeLog
+++ texlive-bin/texk/web2c/omegaware/ChangeLog
@@ -1,3 +1,8 @@
+2015-08-10  Peter Breitenlohner  <peb@mppmu.mpg.de>
+
+	* opl2ofm.ch, ovp2ovf.ch: Send also real numbers to stderr,
+	completing the change from 2015-01-11.
+
 2015-01-18  Peter Breitenlohner  <peb@mppmu.mpg.de>
 
 	* odvicopy.ch: Drop intcast().
--- texlive-bin.orig/texk/web2c/omegaware/opl2ofm.ch
+++ texlive-bin/texk/web2c/omegaware/opl2ofm.ch
@@ -39,6 +39,7 @@
 @y
 @d print(#)==write(stderr,#)
 @d print_ln(#)==write_ln(stderr,#)
+@d print_real(#)==fprint_real(stderr,#)
 @z
 
 @x [still 2] No banner unless verbose.
--- texlive-bin.orig/texk/web2c/omegaware/ovp2ovf.ch
+++ texlive-bin/texk/web2c/omegaware/ovp2ovf.ch
@@ -28,6 +28,7 @@
 @y
 @d print(#)==write(stderr,#)
 @d print_ln(#)==write_ln(stderr,#)
+@d print_real(#)==fprint_real(stderr,#)
 @z
 
 @x [2] Print the banner later.
--- texlive-bin.orig/texk/web2c/pltotf.ch
+++ texlive-bin/texk/web2c/pltotf.ch
@@ -32,6 +32,7 @@
 @y
 @d print(#)==write(stderr,#)
 @d print_ln(#)==write_ln(stderr,#)
+@d print_real(#)==fprint_real(stderr,#)
 @z
 
 @x [still 2] No banner unless verbose.
--- texlive-bin.orig/texk/web2c/vptovf.ch
+++ texlive-bin/texk/web2c/vptovf.ch
@@ -21,6 +21,7 @@
 @y
 @d print(#)==write(stderr,#)
 @d print_ln(#)==write_ln(stderr,#)
+@d print_real(#)==fprint_real(stderr,#)
 @z
 
 @x [2] Print the banner later.
